src/infrastructure/persistence/Migrator.kt

```diff
@@ -10,15 +12,17 @@ interface Migrator {
 class FlyWayMigrator(
     private val configuration: StorageConfiguration,
     private val historyTableName: String = "migration_versions",
-    private val migrationsDirectory: String = "/migrations/"
+    private val migrationsDirectory: String = "/migrations/",
+    private val logger: Logger = LoggerFactory.getLogger("app")
 ) : Migrator {
 
     override fun migrate(): Unit = try {
         execute()
     } catch (e: FlywaySqlException) {
         val message = e.message ?: ""
-
+        logger.error(message)
         if (message.contains("Unable to obtain connection")) {
+            logger.info("Retrying database connection")
             Thread.sleep(500)
```

Comment:
Is this contain stable for every possible error it is gonna give (or is this only to fetch the 'unable to obtain connection')?

Reply:
Only for the error with that message. Not that great I agree but didn't want to touch more than I needed to make it work.